test/functional/rpc_net.py

```diff
@@ -104,6 +125,18 @@ def _test_getpeerinfo(self):
         assert_equal(peer_info[1][0]['addrbind'], peer_info[0][0]['addr'])
         assert_equal(peer_info[0][0]['minfeefilter'], Decimal("0.00000500"))
         assert_equal(peer_info[1][0]['minfeefilter'], Decimal("0.00001000"))
+        # check the `servicesnames` field
+        for info in peer_info:
+            if int(info[0]["services"]) & NODE_NETWORK:
```

Comment:
Could move this into a function for less duplication?

```py
assert_network_service_names(services=int(info[0]["services"]), names=info[0]["servicesnames"])

Reply:
Done